### PR TITLE
Appender benchmark

### PR DESCRIPTION
**Pull request by szarnyasg**
Draft PR, needs the DuckDB JDBC driver with the appender code. This is currently residing on [my fork](https://github.com/szarnyasg/duckdb/tree/appender-jni).

### CODE REVIEW

File: benchmarks/src/main/java/org/eclipse/hawk/duckdb/benchmarks/load/AppenderBenchmark.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	public AppenderBenchmark(DataGenerator gen, int nRows) {
+		super(gen, nRows, false);
+	}
+
+	@Override
+	protected void loadData() throws SQLException {
+		DuckDBConnection conn = (DuckDBConnection) duckDB;
+		try (DuckDBAppender appender = conn.createAppender("main", "data")) {
+			for (int i = 0; i < nRows; i++) {
+				if (autocommit && i % 100 == 0) {
```
Comment (agarciadom) [inline benchmarks/src/main/java/org/eclipse/hawk/duckdb/benchmarks/load/AppenderBenchmark.java]: You can remove this `if`: autocommit is going to be `false` anyway :-).

Comment (szarnyasg) [reply benchmarks/src/main/java/org/eclipse/hawk/duckdb/benchmarks/load/AppenderBenchmark.java]: Thanks, fixed.

**Comment by szarnyasg** [comment]
DuckDB v0.2.4 is in Maven Central now, so this should be good to go.

**Comment by szarnyasg** [comment]
Up

**Pull request merged**